app/models/form.rb

```diff
@@ -17,9 +17,7 @@ def start_page
   end
 
   def has_draft_version
-    return true if made_live_forms.blank?
-
-    updated_at > live_at
+    !live?
```

Comment:
I think it would be better to be more explicit with this logic; while it is true currently that there are only three states and only one of those doesn't have a draft, that won't always be true. I think listing out the states that have draft versions will make it easier to update this logic in future.

```suggestion
    draft? || draft_with_live?
```